src/wlapplication_messages.cc

```diff
@@ -49,70 +53,76 @@ void fill_parameter_vector() {
 	  {"", _("widelands <save.wgf>/<replay.wrpl>"), "--", "", false},
 	  /// Paths
 	  {_("Options:"), "datadir", _("DIRNAME"),
-		_("Use specified directory for the widelands data files"), false},
+		_("Use the specified directory for the widelands data files."), false},
 	  {"", "homedir", _("DIRNAME"),
-		_("Use specified directory for widelands config files, savegames and replays. Default is ") +
-		   kDefaultHomedir,
-		false},
-	  {"", "localedir", _("DIRNAME"), _("Use specified directory for the widelands locale files"),
+		format(_("Use the specified directory for widelands config files, savegames and replays. "
+		         "Default is `%s`."),
+		       kDefaultHomedir),
 		false},
+	  {"", "localedir", _("DIRNAME"),
+		_("Use the specified directory for the widelands locale files."), false},
 	  {"", "language",
 		/** TRANSLATORS: The … is not used on purpose to increase readability on monospaced terminals
 		 */
-		_("[de_DE|sv_SE|...]"), _("The locale to use"), false},
+		_("[de_DE|sv_SE|...]"), _("The locale to use."), false},
 	  {"", "skip_check_datadir_version", "",
-		_("Do not check whether the data directory to use is compatible with this Widelands version"),
+		_("Do not check whether the data directory to use is "
+		  "compatible with this Widelands version."),
 		true},
 	  /// Game setup
 	  {"", "new_game_from_template", _("FILENAME"),
-		_("Directly create a new singleplayer game configured in the given file. An example can be "
-		  "found in `data/templates/new_game_template`"),
+		format(_("Directly create a new game with the settings configured in the given file. "
```

Comment:
I think (but English isn't my native language) that directly would be better after "new game". (affects quite a few options)